Automate dishonest host mitigation (#10)
foranetworks/protocol: core/core.md

However, Hosts are not accountable to community governance and thus Fora does not assume that Hosts will be honestly following the protocol. So how does Fora intend to protect honest members from dishonest Hosts who refuse to comply with community decisions?

> The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a list of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts illustrated in the following scenarios away from the user providing a seamless experience.

Reviewer: ```suggestion
The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a list of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts while providing a seamless experience for the user.
```

Reviewer: Had to replicate @colin-axner suggestion since it became outdated

> The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a lists of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts illustrated in the following scenarios away from the user providing a seamless experience.

Reviewer 2: ```suggestion
The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a list of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts illustrated in the following scenarios away from the user providing a seamless experience.
```

> The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a lists of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts illustrated in the following scenarios away from the user providing a seamless experience.

Reviewer 2: ```suggestion
The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a lists of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts while providing a seamless experience for the user.
```

Not sure how to adjust it, but it was a little hard to read

Reviewer: Going with @colin-axner 's suggestion here, since it isn't just the following scenarios but any form of dishonesty that can be either hidden or obviously attributable with well-written light client code. If there's any meaning lost here, @hamdiallam please open a new pr.

> The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a lists of hosts to connect to, similar to a the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts illustrated in the following scenarios away from the user providing a seamless experience.

Reviewer: ```suggestion
The following will illustrate scenarios in which a host can act dishonestly. While the solutions to these scenarios seem manual, light client code should always have a lists of hosts to connect to, similar to the options provided by a DNS query. This way, light clients can completely automate the detection and mitigation of dishonest hosts illustrated in the following scenarios away from the user providing a seamless experience.
```
